src/OSPSuite.Core/Domain/PKAnalyses/PKParameter.cs

```diff
@@ -33,10 +33,10 @@ public virtual string DisplayName
 
       public virtual PKParameterMode Mode { get; set; }
 
-      public virtual Unit DisplayUnit
+      public virtual string DisplayUnitName
```

Comment:
In order to support any possible units, we need to make sure we have a string here and not a unit. If this does not exist, we will export values in their default unit